Set_movable_separability_2/test/Set_movable_separability_2/CMakeLists.txt

```diff
@@ -18,19 +17,20 @@ set(CMAKE_CXX_STANDARD_REQUIRED TRUE)
 
 find_package(CGAL QUIET COMPONENTS )
 
-if ( CGAL_FOUND )
-
+if (CGAL_FOUND)
   include( ${CGAL_USE_FILE} )
-
   include( CGAL_CreateSingleSourceCGALProgram )
-
   include_directories (BEFORE "../../include")
-
   create_single_source_cgal_program( "test_top_edges_single_mold_trans_cast.cpp" )
   create_single_source_cgal_program( "test_is_pullout_directions_single_mold_trans_cast.cpp" )
 
 else()
-
-    message(STATUS "This program requires the CGAL library, and will not be compiled.")
-
+  if (RUNNING_CGAL_AUTO_TEST)
+    # Just to suppress a warning from CMake when that variable is set on the command line...
+  endif()
+  if (CGAL_DIR)
+    # Just to suppress a warning from CMake when that variable is set on the command line...
+  endif()
```

Comment:
[The CMake warnings are still threre][1]. The dummy uses of `RUNNING_CGAL_AUTO_TEST` and `CGAL_DIR` should be before the `return()` [line 11][l11].

[1]: https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-4.12-Ic-30/Set_movable_separability_2/TestReport_lrineau_CentOS6-32.gz
[l11]: https://github.com/CGAL/cgal/pull/2259/commits/99a673fd0e2d4b345037bdd070fb2b049fe3c71e#diff-89ea674154596b8f9e8a91734fee530dR11